tests/ops/api/v1/endpoints/test_privacy_request_endpoints.py

```diff
@@ -1369,6 +1369,59 @@ def test_get_privacy_requests_sets_days_left(
         data = response.json()["items"][0]
         assert data["days_left"] == days_left
 
+    @mock.patch(
+        "fidesops.ops.service.privacy_request.request_runner_service.run_privacy_request.delay"
+    )
+    def test_sort_privacy_request_by_due_date(
+        self,
+        run_access_request_mock,
+        generate_auth_header,
+        url,
+        db,
+        api_client: TestClient,
+        policy: Policy,
+    ):
+        days_left_values = []
+        data = []
+        now = datetime.utcnow()
+        for i in range(0, 10):
+            requested_at = now + timedelta(days=i)
+            data.append(
+                {
+                    "requested_at": str(requested_at),
+                    "policy_key": policy.key,
+                    "identity": {"email": "test@example.com"},
+                }
+            )
+            days_left_values.append(i + policy.execution_timeframe)
+
+        api_client.post(url, json=data)
```

Comment:
It's worth noting that this test won't differentiate effectively whether you're sorting on `due_date` or `created_at`, since they're effectively the same order (created_at descending will return the same order as due_date descending).

Adding an element of randomness to the due date would fix that.

Reply:
True, good find. I'll update the test to not be the same as `created_at`